Gruntfile.js

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2013-2014 OCAD University
```

Comment:
Is this Copyright correct? Was this file copied from somewhere else, or does it need to be updated/changed?

Reply:
It was copied, and I changed it.  I didn't add copyright headers to the other files we created.  Is there a plugin and/or template we use in general that I can reuse here, or is it just a copy and paste for each file?